cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -95,4 +104,6 @@ function up() {
         echo "Waiting for cluster components..."
         sleep 5
     done
+
+    copy_istio_cni_conf_files
```

Comment:
Please add the previous comment and links to tracking issue.

Reply:
FWIU changes should be exactly as before, initially introduced by https://github.com/kubevirt/kubevirtci/pull/932

Reply:
added them again